README.md

```diff
@@ -148,6 +146,22 @@ Since mailing action methods is based on instance methods defined in a mailer cl
 - If not, all the params in the mailing action method will be T.untyped.
 - For return type though, the mailing action method will return `ActionMailer::MessageDelivery` instead of the return type of the instance method.
 
+## Features Provided at Runtime
```

Comment:
I think we should change the current "Type-checking Rails code" to "Static RBI Generation", and move the documentation of run-time features there to this section.

Reply:
What's needed for this section to work properly is running `rake rails_rbi:custom`, which will copy the rbi for the provided methods.